data/World/Dodongos Cavern MQ.json

```diff
@@ -109,12 +180,22 @@
                 has_bottle and (
                     is_adult or has_explosives or can_use(Dins_Fire) or dodongos_cavern_shortcuts or
                     (Sticks or ((Nuts or Boomerang) and (Kokiri_Sword or Slingshot)))
-                )"
+                )",
+            "Dodongos Cavern MQ Room Before Boss Pot 1": "True",
+            "Dodongos Cavern MQ Room Before Boss Pot 2": "True",
+            "Dodongos Cavern MQ Armos Army Room Pot 1" : "
+                is_adult or has_explosives or Boomerang or can_use(Dins_Fire) or
+                (dodongos_cavern_shortcuts and (can_child_attack or Progressive_Strength_Upgrade)) or
+                (Sticks or ((Nuts or Boomerang) and (Kokiri_Sword or Slingshot)))",
```

Comment:
Looks like you stole the logic for this armos army room pot from the nearby skulltula, rather than the fairy pot right next to it. Use the fairy pot logic (minus the bottle).

I'm not okay with the numbering 1, 2, 3 here, when one of the pots has such different requirements from the others.